include/JSystem/JFW/JFWDisplay.h

```diff
@@ -51,26 +55,28 @@ struct JFWDisplay {
 	void clearAllXfb();
 	void frameToTick(f32);
 
+	static JFWDisplay* getManager() { return sManager; }
+
 	// _00 VTBL
 	JUTFader* m_fader;    // _04
 	JUtility::TColor _08; // _08
 	u32 _0C;              // _0C
 	JUTXfb* m_Xfb;        // _10
 	u16 _14;              // _14
-	u32 _18;              // _18
+	int m_drawDoneMethod; // _18
 
 	/// PikDecomp calls this "fpsDivider".
 	u16 m_secondsPer60Frames; // _1C
 
-	u32 _20;  // _20
-	bool _24; // _24
-	u16 _26;  // _26
-	f32 _28;  // _28
-	int _2C;  // _2C
-	int _30;  // _30
-	int _34;  // _34
-	u16 _38;  // _38
-	u8 _3A;   // _3A
+	u32 m_tickRate;         // _20
+	bool m_enableAlpha;     // _24
```

Comment:
When naming booleans, please make sure we're asking the question, in this example it'll be something like `mIsAlphaEnabled`